controllers/sponsor.controller.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+const Services = {
+    Sponsor: require("../services/sponsor.service"),
+    Logger: require("../services/logger.service")
+};
+const Util = require("../middlewares/util.middleware");
+
+/**
+ * @async
+ * @function findById
+ * @param req
+ * @param res
+ * @return {JSON} Success or error status
+ * @description Retrieves a sponsor's information via it's mongoId specified in req.params.id
+ */
+async function findById(req, res) {
+    // finds sponsor by route parameter
+    const sponsor = await Services.Sponsor.findById(req.params.id);
+
+    if (sponsor) {
+        return res.status(200).json({
+            message: "Successfully retrieved sponsor information",
+            data: sponsor.toJSON()
```

Comment:
I think that if someone has the appropriate permissions to retrieve the sponsor information they should be able to retrieve all of it. For example, if a sponsor wanted to see their own information, I don't think any of accountId, tier, company, contractURL, or nominees need to be hidden. I'm just not sure how auth/permissions work right now, so that's not included in the middleware chaining.